server/jetstream.go

```diff
@@ -146,10 +146,10 @@ const (
 	JetStreamUpdateMsgWithRevision = "$JS.%s.UPDATE"
 
 	// JetStreamObservableAckSamplePre is the prefix for sample messages from observables
-	JetStreamObservableAckSamplePre = "$JS.EVENTS.OBSERVABLE.ACKSAMPLE"
+	JetStreamObservableAckSamplePre = "$JS.SAMPLE.OBS_ACK"
```

Comment:
I think this is also an event.

$JS.EV.CONSUMER_ACK_METRIC

That may need to be shortened but hopefully makes sense. If we want to hoist METRIC as its own thing..

$JS.METRIC.ACK.<stream>.<consumer>